exercises/practice/gigasecond/test/gigasecond_test.dart

```diff
@@ -2,49 +2,19 @@ import 'package:gigasecond/gigasecond.dart';
 import 'package:test/test.dart';
 
 void main() {
-  group("Gigasecond", gigasecondTests);
-}
-
-void gigasecondTests() {
-  group('Add one gigasecond to the input. ', () {
-    test('date only specification of time', () {
-      final DateTime birthDate = DateTime.utc(2011, DateTime.april, 25);
-      final DateTime result = add(birthDate);
-      DateTime expectedDate = DateTime.utc(2043, DateTime.january, 1, 1, 46, 40);
-
-      expect(result, equals(expectedDate));
-    }, skip: false);
-
-    test('second test for date only specification of time', () {
-      final DateTime birthDate = DateTime.utc(1977, DateTime.june, 13);
-      final DateTime result = add(birthDate);
-      DateTime expectedDate = DateTime.utc(2009, DateTime.february, 19, 1, 46, 40);
-
-      expect(result, equals(expectedDate));
-    }, skip: true);
-
-    test('third test for date only specification of time', () {
-      final DateTime birthDate = DateTime.utc(1959, DateTime.july, 19);
-      final DateTime result = add(birthDate);
-      DateTime expectedDate = DateTime.utc(1991, DateTime.march, 27, 1, 46, 40);
-
-      expect(result, equals(expectedDate));
-    }, skip: true);
-
```

Comment:
I would argue that `date`, `date-time` are hard concepts to wrap your brain around. Every single tool/language does it differently. There are standards which have inconsistencies in them and then when they are implemented they are not standards compliant.

While `gigasecond` is not specifically about `date`, `date-time`.  And while the goal of an exercise is more about being fun to solve. 

I think we should keep these tests as they are specifically trying to make sure the correctness of `gigasecond` _in Dart_.

What kind of shenanigans does Dart's date, date-time implementation have? I don't remember. But would I want my date, date-time related code to work as expected in a real world? You bet!.

Since dart can be compiled to web (JavaScript), does it affect result of this `gigasecond` implementation? Does Dart compile it's own implementation of DateTime or does it use the one provided by browser implementation? I don't really care!. I mostly care that it works as expected.

I think the date only tests could get a better description than just saying `second test...` or `third test...`, that would've helped justify their existence better.


P.S. Although it may seem like it, my tone is _not_ sarcastic. It's more playful, but after reading my comment I deemed it necessary to clarify.

Reply:
I see where you're coming from, but the tests don't seem to accomplish any of this. They simply define a `DateTime` with no hours, minutes, or seconds. Does this really help explore the problem space?

Perhaps we could invent a new exercise that really does accomplish the above?

Reply:
> Does this really help explore the problem space?

I thought about. I could not convince myself. So my answer is no.

> Perhaps we could invent a new exercise that really does accomplish the above?

I think that's the better approach, since `gigasecond` is really a one-liner exercise. One of the easy ones. Does Exercism have exercises with the theme of "common pitfalls"? This could fall under that.

Reply:
Ok, excellent. We don't have concept-exercises that are "common pitfalls" as such, but there certainly are ways of addressing common pitfalls as part of exercises.

If you are feeling inspired to tackle something, go ahead and open an issue about it and we can figure out how to narrow it down to an actual exercise.